packages/sdk-rtl/README.md

```diff
@@ -7,3 +7,13 @@ The source code in this package is almost all completely generic REST request/re
 The `@looker/sdk` package is updated with every Looker release. This package has a much longer update/release cycle.
 
 **DISCLAIMER**: This is a _beta_ version of the Looker SDK runtime library. Implementations are still subject to change.
+
+## Using jest tests with this SDK as a dependency
+
+This jest config entry may be required to run jest tests on projects which depend on this package.
+
+```js
+ transformIgnorePatterns: [
+    '<rootDir>/node_modules/(?!@looker/(sdk-rtl|sdk)/.*)',
+  ],
+```
```

Comment:
Apologies. Forgot to mention this. They may need to update the transform to include js files
```
  transform: {
    '^.+\\.(js|ts|tsx)?$': 'ts-jest',
  },
```